sonar/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -20,19 +20,35 @@ msgstr ""
 
 #: sonar/config.py:47
 msgid "French"
-msgstr ""
+msgstr "Français"
 
 #: sonar/config.py:48
 msgid "German"
-msgstr ""
+msgstr "Allemand"
```

Comment:
On RERO ILS, there a pending PR rero/rero-ils#354 that aimed to keep the languages menu in the original languages, ie "French" is always "Français" and "German" always "Deutsch", in order to help the user to select his or her preferred language.

The commit is this one: https://github.com/rero/rero-ils/pull/354/commits/5346d951a7464474b8bb81ad403113590eb2e7ff

Reply:
This translation comes from config.py and is not used in UI at this time. Anyway, I corrected those messages

Reply:
@iGormilhit Ok for you ?